src/components/NoteDetails/NoteDetailsPage.js

```diff
@@ -1,31 +1,33 @@
 import React from 'react';
 
-import IconButton from '@material-ui/core/IconButton';
-import ArrowBackIcon from '@material-ui/icons/ArrowBack';
-import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+import { colors } from 'colors';
 
 import { NoteDetailsContainer } from 'components/NoteDetails/NoteDetailsContainer';
 
-const Wrapper = styled.div`
-  background-color: #2b2630;
-`;
+import NoteDetailsTopBar from './NoteDetailsTopBar';
+import ModalDeleteComponent from 'components/ModalDelete/ModalDeleteComponent';
+import { useState } from 'react';
 
-const BackButtonWrapper = styled.div`
-  color: red !important;
-  padding: 1rem;
+const Wrapper = styled.div`
+  background-color: ${colors.darkViolet};
 `;
 
 export const NoteDetailsPage = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
   return (
     <Wrapper>
-      <BackButtonWrapper>
-        <Link to="/notes">
-          <IconButton aria-label="back">
-            <ArrowBackIcon style={{ fill: '#E4E3E4' }} />
-          </IconButton>
-        </Link>
-      </BackButtonWrapper>
+      <NoteDetailsTopBar handleClickOpen={handleClickOpen} />
+      <ModalDeleteComponent open={open} handleClose={handleClose} />
```

Comment:
I think it would be better to put this modal into `NoteDetailsTopBar` component, since the button opening the modal is there.
And also `changeParent` button click is handled inside `NoteDetailsTopBar`.
I believe it would make more sense to handle all the buttons in `NoteDetailsTopBar`, rather than handling one button in `NoteDetailsTopBar` and another button in `NoteDetailsPage`.